Code/Tools/LuaIDE/Source/Editor/LuaEditor.cpp

```diff
@@ -6,13 +6,8 @@
  *
  */
 
-#include "LuaEditor.h"
 #include <Source/LuaIDEApplication.h>
```

Comment:
Did the Lua IDE compile with this removed?

Reply:
yes, there are some small things to fix in other platforms, but windows compiled and tests passed: https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE/detail/PR-5532/1/pipeline/295